lib/inferno/dsl/http_client.rb

```diff
@@ -60,16 +60,17 @@ def http_clients
       # @param client [Symbol]
       # @param name [Symbol] Name for this request to allow it to be used by
       #   other tests
-      # @param _options [Hash] TODO
+      # @param options [Hash] Input headers here - headers are optional and
```

Comment:
You should use `@option` to document an options hash: https://www.rubydoc.info/gems/yard/file/docs/Tags.md#option